File: aspnet-core/src/emes.Application/Services/MasterData/MaterialAssignment/Dto/MaterialAssignmentDto.cs


namespace emes.Services.MasterData.MaterialAssignment.Dto
{
    [AutoMapFrom(typeof(MaterialAssignmentModel))]
    public class MaterialAssignmentDto : EntityDto<Guid>
    {
        public string MaterialNumber { get; set; }
        public string Description { get; set; }
        public int MaterialStatus { get; set; }
        public string MaterialType { get; set; }
    }
}

File: reactjs/src/services/materialAssignment/Dto/MaterialAssignmentDto.ts
export interface GetAllAssignmentInput{
   id:number,
   materialNumber:string,
   materialStatus: number,
   materialType:string
   description:string
}

export interface GetAllAssignmentOutput{
   id:number,
   materialNumber:string,
   materialStatus: number,
   materialType:string
   description:string
}

export interface createInput{
   id:number,
   materialNumber: string,
   organizationUnitId :number
}
export interface createForAdmin{
   id:number,
   materialNumber: string,
   organizationUnitId :number
}

